### DISCUSSION TITLE
Setting Access keys on a NavigationView's Settings and Back buttons?

### DISCUSSION THREAD

**Discussion by DHancock** [microsoft/microsoft-ui-xaml#7447] [Q&A]
I'm trying to set the access key for a NavigationView's Settings button but cannot find a way to do this. I saw issue #941 which details the same problem but it has been closed recommending that the work around contained in the issue is documented. I found the work around documented [here](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.navigationview?view=winui-3.0#remarks) but it no longer works because, perhaps not surprisingly the internal structure of the NavigationView has subsequently changed (and may change again in the next release).

My question is "is there now a prefered way to set access keys and key tip placement mode for the Settings and Back buttons?"

(I'm using the Microsoft.ProjectReunion nuget packages version 0.8.2 in a win32 desktop app. The NavigationView's  PaneDisplayMode is set to Top)

**Comment by DHancock** [comment]
With a bit of digging I found that I can simply hide the NavigationView's built in settings button and replace it with my own placed inside the _NavigationView.FooterMenuItems_. I'm using the NavigationView just as a tab menu which doesn't need a back button though. The back button is a different problem...

**Comment by StephenLPeters** [comment]
@chigy looks like our documentation is out of date https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.navigationview?view=winui-3.0#remarks.  @karenbtlai and @ojhad any insights on doing this not that NavView is using repeater?
